docs/snippets/CMakeLists.txt

```diff
@@ -117,6 +117,26 @@ target_link_libraries(${TARGET_NAME} PRIVATE openvino::runtime)
 
 # [cmake:integration_example_cpp]
 
+set(TARGET_NAME_PY "ov_integration_snippet_py")
```

Comment:
could we also add to this place https://docs.openvino.ai/2024/documentation/openvino-extensibility.html#create-a-library-with-extensions ?

Reply:
added there too